provisioning/common/etcd/values.yaml

```diff
@@ -252,7 +252,7 @@ extraDeploy:
                           name: {{ include "etcd.secretName" . }}
                           key: {{ include "etcd.secretPasswordKey" . }}
                     {{- end }}
-                    {{- if .Values.defragmentation.cronjob.resources.limits.memory }}
+                    {{- if ((.Values.defragmentation.cronjob.resources).limits).memory }}
```

Comment:
`.Values.defragmentation.cronjob.resources` je nil a preto sa nemozem pytat na `.limits.memory`.

Nasiel som na to takuto fintu, ... v go `text/template`, ked je hodnota v zatvorke `nil`, tak sa to dalej nevyhodnocuje.